ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml

```diff
@@ -71,7 +71,8 @@ spec:
 
         if [ -f $RESULTS_FILE ]; then
           echo "Results exist"
-          echo -n "true" | tee $(results.preflight_results_exists.path)
+          CHECK_RESULTS=$(cat $RESULTS_FILE | jq -r '.passed')
```

Comment:
Please add a log message to tell the user if these results passed.

Reply:
Updated!